tstest/integration/testcontrol/testcontrol.go

```diff
@@ -172,6 +173,19 @@ func (s *Server) serveMachine(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+// servePingInfo currently placeholder, determine the correct response to client
```

Comment:
Let's wait until we have something more than a placeholder to actually submit this code. (Sorry for misleading you over chat earlier, I thought this was a completed handler.)

Reply:
sure, ill let you know once the contents are determined